From: Tommy Webb <tommy@calyxinstitute.org>
Date: Tue, 4 Mar 2025 19:13:05 +0000
Subject: Fix build after "Add bookmark import/export actions"

Make a couple of methods public, since they were internal and not
accessible for use by Cromite.

Change-Id: Ia0a18b8dd8472eb9690a124fb715e8b8da40d7e3
---
 .../browser/download/dialogs/DownloadLocationCustomView.java  | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java
--- a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java
+++ b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java
@@ -85,7 +85,7 @@ public class DownloadLocationCustomView extends ScrollView
         mDirectoryAdapter.update();
     }
 
-    void setTitle(CharSequence title) {
+    public void setTitle(CharSequence title) {
         mTitle.setText(title);
     }
 
@@ -93,7 +93,7 @@ public class DownloadLocationCustomView extends ScrollView
         mSubtitleView.setText(subtitle);
     }
 
-    void setFileName(CharSequence fileName) {
+    public void setFileName(CharSequence fileName) {
         mFileName.setText(fileName);
     }
 
-- 

